src/client/conn/http2.rs

```diff
@@ -240,22 +257,22 @@ where
 
 // ===== impl Builder
 
-impl Builder {
+impl<Ex> Builder<Ex>
+where
+    Ex: Executor<BoxSendFuture> + Send + Sync + 'static + Clone,
```

Comment:
This bounds likely prevents using a local executor. Could probably remove it.

Reply:
It only requires Clone now